code/constness/constplay.cpp

```diff
@@ -1,23 +1,23 @@
 #include <iostream>
 #include <string>
 
-int identity(int a) {
-    return a;
+void copy(int a) {
+    [[maybe_unused]] int val = a;
```

Comment:
```suggestion
/* This is a dummy function to demonstrate pass by value.
 * Since it doesn't do anything with the argument, we suppress
 * possible compiler warnings using `maybe_unused`.
 */
void copy(int a) {
    [[maybe_unused]] int val = a;
```